components/clarity-repl/src/repl/interpreter.rs

```diff
@@ -116,54 +118,93 @@ impl ClarityInterpreter {
         }
     }
 
-    pub fn run(
+    pub fn run_both(
         &mut self,
         contract: &ClarityContract,
+        ast: &mut Option<ContractAST>,
         cost_track: bool,
         eval_hooks: Option<Vec<&mut dyn EvalHook>>,
     ) -> Result<ExecutionResult, Vec<Diagnostic>> {
-        let (mut ast, mut diagnostics, success) = self.build_ast(contract);
+        let result = self.run(contract, ast, cost_track, eval_hooks);
+
+        // when running clarity-repl/wasm (ie. not natively), we can't run clar2wasm
+        #[cfg(all(feature = "cli", not(feature = "wasm")))]
+        if self.repl_settings.is_clarity_wasm_enabled() {
+            let mut contract_wasm = contract.clone();
+            contract_wasm.deployer =
+                ContractDeployer::Address("ST3NBRSFKX28FQ2ZJ1MAKX58HKHSDGNV5N7R21XCP".into());
```

Comment:
What happens if this address isn't funded? Do we always fund it?

Reply:
Good point. It depends on the wallets in the devnet config. I'll add a comment about that in #1306